packages/foreman/rubygem-sqlite3/rubygem-sqlite3.spec

```diff
@@ -77,6 +77,13 @@ cp -a .%{gem_extdir_mri}/%{gem_name}/*.so %{buildroot}%{gem_extdir_mri}/%{gem_na
 # Prevent dangling symlink in -debuginfo (rhbz#878863).
 rm -rf %{buildroot}%{gem_instdir}/ext/
 
+%check
+%if 0%{rhel} != 8
```

Comment:
@zjhuntin @pcreech this is really a workaround to allow me to continue this PR, but I think this package is broken on EL8.